devicetypes/smartthings/zll-rgb-bulb.src/zll-rgb-bulb.groovy

```diff
@@ -1,5 +1,5 @@
 /**
- *  Copyright 2017 SmartThings
+ *  Copyright 2017-2019 SmartThings
```

Comment:
Best not to put an end date. Without it the copyright continues to the current date (I think)

Reply:
Hmmmm Maybe that is what I had heard. Though, a quick look seems to indicate that really none of it matters? But, I'll revert that commit, and fix the couple of other code formatting things that I missed.